sui/deploy-contract.js

```diff
@@ -269,16 +271,17 @@ async function upgrade(keypair, client, supportedPackage, policy, config, chain,
     const contractConfig = contractsConfig?.[packageName];
 
     validateParameters({ isNonEmptyString: { packageName } });
+    const moveDir = `${__dirname}/move`;
 
     if (packageDependencies) {
         for (const dependencies of packageDependencies) {
             const packageId = contractsConfig[dependencies]?.address;
-            updateMoveToml(dependencies, packageId);
+            updateMoveToml(dependencies.toLowerCase(), packageId, moveDir);
```

Comment:
updateMoveToml is not the most elegant to read, can you refactor it to use a toml parser so it cleanly reads/writes the file?

Reply:
will do this in a separate pr